### ISSUE TITLE
No exported TypeScript interface

### ISSUE THREAD

**Issue by BenJackGill** [ps-cristopher/vue3-autocounter#8]
We need an exported TypeScript interface named something like `Vue3Autocounter` that we can import and use for the [TypeScript template ref definition](https://vuejs.org/guide/typescript/composition-api.html#typing-template-refs).

Currently there is no exported interface for us to use. Or at least none that I could find.

For example:

```TypeScript
<template>
  <vue3autocounter ref='counter' :startAmount='0' :endAmount='2021' :duration='3' '/>
  <button @click='handleClick'>Start counting</button>
</template>

<script setup lang="ts">
// Import everything
import Vue3autocounter from "vue3-autocounter";

// Define the template ref
const counter = ref<Vue3Autocounter | null>(null);

// Call the start method
const handleClick = () => {
  counter.start();
}
</script>
```

Here I am using `Vue3Autocounter` in `const counter = ref<Vue3Autocounter | null>(null)` as an example. But actually that example will not work because `Vue3Autocounter` was just made up for illustration purposes and was never imported because it doesn't currently exist.

**Comment by ps-cristopher** [comment]
You are right, in the last released version is not possible to have an exported type for the component. I`m working on the next release with no breaking changes and I'll add this as an enhancement in this new version. You can open a PR with the implementation if you need it now, in the meantime.

Thanks, @BenJackGill
